core/src/main/scala/cats/data/Kleisli.scala

```diff
@@ -21,6 +21,9 @@ final case class Kleisli[F[_], A, B](run: A => F[B]) { self =>
   def map[C](f: B => C)(implicit F: Functor[F]): Kleisli[F, A, C] =
     Kleisli(a => F.map(run(a))(f))
 
+  def contramap[C](f: C => B)(implicit F: Contravariant[F]): Kleisli[F, A, C] =
+    Kleisli(a => F.contramap(run(a))(f))
+
```

Comment:
For simplicity, I'd be inclined to just keep the one until someone runs into a compelling use-case for the other.